library/core/proguard-rules.txt

```diff
@@ -58,6 +58,16 @@
   <init>(com.google.android.exoplayer2.upstream.DataSource$Factory);
 }
 
+-dontnote com.google.android.exoplayer2.extractor.SeekPoint
+-keepclasseswithmembers class com.google.android.exoplayer2.extractor.SeekPoint {
+  <init>(long, long);
+}
+
+-dontnote com.google.android.exoplayer2.extractor.SeekMap$SeekPoints
+-keepclasseswithmembers class com.google.android.exoplayer2.extractor.SeekMap$SeekPoints {
+  <init>(com.google.android.exoplayer2.extractor.SeekPoint, com.google.android.exoplayer2.extractor.SeekPoint);
+}
```

Comment:
I think this configuration should go in the flac extension rather than the core library, since these only need to be retained if the extension is being used.